SU-HSDO/suhumsci
# STN-738: Add Photo Album Slider counter

# READY FOR REVIEW

## Summary
As a site user, I want a counter to display on the Photo Album slider that will indicate what slide and how many slides total are in the Photo Album. The Photo Album slider will display up to 30 slides. The counter styles must work with caption and credits as well as any slider UI controls.

Note that the implementation details in the [JIRA card STN-738](https://sparkbox.atlassian.net/browse/STN-738) provide implementation details outlining the use of SlickJS and JavaScript to add the photo album slider counter. After much investigation it was discovered that JavaScript was not necessary and that this work could be accomplished using CSS. :tada:

## Example Screenshot

![photo album slider](https://user-images.githubusercontent.com/12678977/123702524-3dfe3880-d831-11eb-8645-f092659b0b43.png)

## Need Review By (Date)
asap!

## Urgency
:fire:

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests continue to pass.
2. Next run `lando drush @sparkbox_sandbox.local cr` to clear the cache and ensure that you are seeing the updated configuration file and styles.
3. To review the work in this PR you will need to use a page which has the photo album component. If you have synced with staging recently you can use https://sparkbox-sandbox-stage.stanford.edu/photo-carousel. Note that you may need to update the images within the photo album slider as many appear to be missing on staging. Or you can create a new flexible page and add a photo album slider. [Components > Photo Album. Under `Style` choose `Display Mode of Slideshow`.]  I recommend creating a slider with between 20 to 30 images. Add a mix of vertical and horizontal images.
4. Review the following:
    - [x] The slider counter should visually match the [Figma](https://www.figma.com/file/0UFh6PUAFO4YKazUECfzq3/WEST-Photo-Album-Slider-Counter?node-id=1%3A34) designs.
    - [x] The counter does not interfere with caption and credits on images.
    - [x] The counter does not interfere with the slider navigation controls (arrows).
    - [x] The counter does not interfere with keyboard accessibility.
    - [x] The counter works and displays correctly when logged in and logged out.
    - [x] The counter styles are the same on Colorful and Traditional themes.
    - [x] The counter does not display when there is only one slide in the photo album slider.
    - [x] The work in this PR does not impact or change the photo album grid or other components such as the gradient hero slider or hero overly carousel.
